n2y/notion.py

```diff
@@ -375,13 +381,67 @@ def save_file(self, content, page, extension):
             temp_file.write(content)
         return urljoin(self.media_url, relative_filepath)
 
-    def append_block_children(self, block_id, children):
-        response = requests.patch(
-            f"{self.base_url}blocks/{block_id}/children",
-            json={"children": children}, headers=self.headers
+    def append_block_children(self, block_id, children, page_or_database_info=None):
```

Comment:
Per our call:

Rewrite to remove the need for `page_or_database_info`.

If child pages work, keep that.

If child database don't work, remove the code for this and raise NotImplementedError

Reply:
Rename to `append_notion_blocks`

Reply:
Add a note to the doctoring saying that not all blocks allow you to add children.

Reply:
great news, appending both child_databases and child_pages works perfectly with this new method!